test/RWR/test_rwr.py

```diff
@@ -0,0 +1,69 @@
+import shutil
+from pathlib import Path
+
+import pytest
+
+import spras.config as config
+from spras.rwr import RWR
+
+config.init_from_file("config/config.yaml")
+
+TEST_DIR = 'test/RWR/'
+OUT_FILE_DEFAULT = TEST_DIR+'output/rwr-edges.txt'
+OUT_FILE_OPTIONAL = TEST_DIR+'output/rwr-edges-optional.txt'
```

Comment:
Rename OPTIONAL to OPTIONS or OPTIONAL_ARGS - I think that's what you mean (the optional arguments)